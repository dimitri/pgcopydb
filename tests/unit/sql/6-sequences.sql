select last_value from identity_table_id_seq;
select last_value from normal_table_id_seq;
