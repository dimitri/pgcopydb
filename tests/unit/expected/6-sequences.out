-[ RECORD 1 ]---
last_value | 667

-[ RECORD 1 ]---
last_value | 667

